### ISSUE TITLE
new Geotag ninja release cant be upgraded without first uninstalling existing one

### ISSUE THREAD

**Issue by pbranly** [nemethviktor/GeoTagNinja#12]
as described in the title
Phil

**Comment by nemethviktor** [comment]
That's me being forgetful more often than not. 
The technical explanation is that the project uses Visual Studio Installer, which is a piece of ... junk but it's kinda bundled with VS itself and is free to use. The problem is that VSI creates MSI files that have to be manually "triggered" at build-time to update some value internally so that Windows Installer doesn't think we're trying to reinstall the same version...it can't just seem to pull it automatically from the manifest.
In other words the project "knows" it's a new version because versioning is just based on the date, so each day the project is built is a different version but VSI is stupid and needs to be told explicitly, which I tend to forget because it's in an obscure place.

**Comment by nemethviktor** [comment]
Can you try it with the current (new) release pls?
Generally as I wrote above this is a recurring problem, the installer I use is s...t and if I forget to manually bump the version in it then this happens. Apologies.

**Comment by pbranly** [comment]
this is working
just one remark that is fully marginal: thr different menus during the upgrade use fonts that are sometimes difficult to read.
Phil

**Comment by pbranly** [comment]
in order to have something clean, I have downloaded the last version on github. but bug is back ! or is it a side effect of parallel versions ?
phil
![2022-10-26_19-08](https://user-images.githubusercontent.com/22018509/198091932-33efb8ef-8e86-49ef-a7ff-6b9064b17435.png)
it says that a version exist and that I have to uninstall it before installing the new one

**Comment by nemethviktor** [comment]
Possibly parallel. The version you tested has the same "major version" but
not the "minor version" (ie both are 0.5.8334 but the 5 digit random number
at the end is different because I've run another build to update the
changelog and readme)

This message has been brought to you by my Huawei P30 Pro and myself. Any
typos blame the phone.
